models/migrations/migrations.go

```diff
@@ -574,18 +573,20 @@ var migrations = []Migration{
 	// v293 -> v294
 	NewMigration("Ensure every project has exactly one default column", v1_22.CheckProjectColumnsConsistency),
 
-	// Gitea 1.22.0 ends at 294
+	// Gitea 1.22.0-rc0 ends at 294
```

Comment:
Should we name it like that? RC0 never existed as far as I'm aware, thought I'm not sure if nothing was published.

Reply:
At least no docker tag: https://hub.docker.com/r/gitea/gitea/tags

Reply:
no docker image, no github release, but rc0 does exist: https://dl.gitea.com/gitea/1.22.0-rc0/

Reply:
ok than it exists in some form at least, ok.